Resources/translations/FOSUserBundle.hu.yml

```diff
@@ -42,10 +46,14 @@ registration:
             A fiókja visszaigazolásához kérjük látogasson el a következő linkre: %confirmationUrl%
 
             Üdvözlettel,
-            the Team.
+            a Csapat.
 
 resetting:
-    check_email: 'Egy e-mail lett küldve a(z) %email% címre. Egy linket tartalmaz, amire rá kell kattintani a jeszó lecseréléséhez.'
+    check_email:  |
+        Egy e-mail elküldésre került. Egy linket tartalmaz, amire kattintva cserélheti le jelenlegi jelszavát.
+        Megjegyzés: Új jelszót csak a következő %tokenLifetime% órában igényelhet.
```

Comment:
Az angol is kicsit hülyén van fogalmazva, de ez arról szólna, hogy az adott link csak adott óráig él. Valami ilyesmi talán jobb lenne: "Ennek az új jelszó igénylésnek az érvényessége %tokenLifetime% óra.
Csak tipp.